tensorflow/go/saved_model.go

```diff
@@ -58,6 +58,9 @@ func LoadSavedModel(exportDir string, tags []string, options *SessionOptions) (*
 		return nil, err
 	}
 	cExportDir := C.CString(exportDir)
+	if len(tags) == 0 {
+		return nil, fmt.Errorf("empty tags is not allowed")
```

Comment:
Failed an internal test: missing an import for fmt in this file.

Reply:
Thanks, fixed!